### DISCUSSION TITLE
Request failed with status code 504 when change the way to show alerts

### DISCUSSION THREAD

**Discussion by n0th1nt0lOOz** [Security-Onion-Solutions/securityonion#4177] [Unsupported Versions]
Hi All,

For current I only can show the alert with "Group By Name, Module" option. When change to "Group By Sensor, Source IP/Port, Destination IP/Port, Name" then it returns "Request failed with status code 504". Really need this filter but I don't find the way to solve that. I very appreciate any help :(

![image](https://user-images.githubusercontent.com/28757980/117911427-f9adec00-b307-11eb-9164-39ae6ba04749.png)

**Comment by ll3N1GmAll** [comment]
Did you ever find an answer to this issue? I am getting this result after having SO running for some time (months). Even when I apply no filters to Hunt I'm getting 504 no matter what I do, which has effectively made Hunt broken and useless.

**Reply by dougburks** [reply]
@ll3N1GmAll It seems like you are experiencing something different than the original poster. Please start a new discussion rather than replying to somebody else's discussion.
